wormchain/x/wormhole/keeper/msg_server_register_account_as_guardian.go

```diff
@@ -50,16 +51,6 @@ func (k msgServer) RegisterAccountAsGuardian(goCtx context.Context, msg *types.M
 		return nil, types.ErrGuardianSetNotFound
 	}
 
-	consensusGuardianSetIndex, consensusIndexFound := k.GetConsensusGuardianSetIndex(ctx)
```

Comment:
will removing this change the amount of gas that's used for the transaction? That could lead to consensus errors. If so we could gate this functionality behind a block height.

Reply:
I'm unsure of that. We could still keep the check in to verify an existing consensus guardian set index exists.